tools/system_libs.py

```diff
@@ -1565,13 +1565,23 @@ def get_include_dir():
     return dirname
 
   @staticmethod
-  def install_header_dir(src_dir, target=None):
+  def install_header_dir(src_dir, target=None, merge=False):
     if not target:
       target = os.path.basename(src_dir)
     dest = os.path.join(Ports.get_include_dir(), target)
-    shared.try_delete(dest)
-    logger.debug('installing headers: ' + dest)
-    shutil.copytree(src_dir, dest)
+    if not merge:
```

Comment:
Perhaps flip the conditions to the condition is in the affirmative (`if merge:`...`)

Reply:
Isn't the the main different between `install_header_dir` and `install_headers` that with the former the entire directory is owned by the port (and therefore deleted before install), whereas the with the later we have to merge in with other headers (so we can't just delete it).

I wonder if just using `install_headers` would work instead of adding this extra arg?